pipelines/defaults.json

```diff
@@ -4,7 +4,9 @@
         "build_url"          : "https://github.com/ibmruntimes/temurin-build.git",
         "build_branch"       : "ibm",
         "pipeline_url"       : "https://github.com/ibmruntimes/ci-jenkins-pipelines.git",
-        "pipeline_branch"    : "ibm"
+        "pipeline_branch"    : "ibm",
+        "installer_url"      : "git@github.ibm.com:runtimes/adoptium-installer.git",
+        "installer_branch"   : "ibm"
```

Comment:
I'm guessing this is allowing us to have more configurable builds for testing out changes?

Reply:
correct